UPGRADE.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading
+
+This document describes future changes that will affect your current resources used
+inside of OpenShift. Each change contains description of the change and information
+when that change will happen.
+
+
+## Origin 1.0.0 / OSE 3.0.0
+
+* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
+  All the newly created Builds will already have the new label, old values will
+  be recognized by OpenShift through the 3.0.x release. Support for these values
+  will be removed in OSE 3.1 and Origin 1.x. See #3502.
```

Comment:
These have to be tied to Origin version numbers first, with the
corresponding OSE number after.  When we don't know what either number will
be, we'll try to lock down.

On Thu, Jul 2, 2015 at 9:56 AM, Maciej Szulik <notifications@github.com>
wrote:

> In UPGRADE.md
> <https://github.com/openshift/origin/pull/3508#discussion_r33779740>:
>
> > @@ -0,0 +1,13 @@
> > +# Upgrading
> > +
> > +This document describes future changes that will affect your current resources used
> > +inside of OpenShift. Each change contains description of the change and information
> > +when that change will happen.
> > +
> > +
> > +## Origin 1.0.0 / OSE 3.0.0
> > +
> > +* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
> > +  All the newly created Builds will already have the new label, old values will
> > +  be recognized by OpenShift through the 3.0.x release. Support for these values
> > +  will be removed in OSE 3.1 and Origin 1.x. See #3502.
>
> Agreed, will update once I fix the integration tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3508/files#r33779740>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift